src/post_process/m_data_output.f90

```diff
@@ -939,6 +1029,49 @@ subroutine s_write_variable_to_formatted_database_file(varname, t_step)
 
     end subroutine s_write_variable_to_formatted_database_file ! -----------
 
```

Comment:
I think @wilfonba removed this feature intentionally. please do not add it back.